webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -56,7 +61,17 @@ let createAPIRequest = ({
         id,
       })
     )
-    .catch(error => requestFailure({ error, id, dispatch }));
+    .catch(error => {
+      const { type } = error;
+      if (type !== ERROR_TO_SHOW) {
```

Comment:
Is this specific only to `Autocomplere`? what about a global error mechanism?

Reply:
when the autocomplete query is incorrect, an error should return from the server,
an error that we want to show to the user. other than that, all other errors should be visible in the console only. Do we have something similar elsewhere? Thanks